frontend/js/actions/ArimaaActions.js

```diff
@@ -139,7 +139,7 @@ var ArimaaActions = {
         actionType: ArimaaConstants.ACTIONS.GAME_SETUP_OVER
       });
     }
-    
+
```

Comment:
I should probably add a save hook to my editor to remove trailing whitespace. :)